### PR TITLE
RFC-1000 Technical Editing

### PR DESCRIPTION
**Pull request by anselld**
Technical editing.
## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [x] Documentation

## Checklist:

* [x] I'm merging against the `development` branch
* [ ] I ran `cargo-fmt --all` before pushing
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by anselld** [comment]
Hi @hansieodendaal , please see my suggested changes. Thanks.

File: RFC/src/RFC-1000_TariUseCases.md [hunk 1]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
-manifested are shown below.
+manifested are provided here:
 
 #### Crystal Skull of Akador
 
-* Is rare
-	* Is 1 of 5
+* Is rare, it is 1 of 5.
+
 * Is legendary
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
* Is legendary:
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 2]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
-	* If you have this item while visiting Indiana Jones World, you get to skip the line three times.
-* Is a contingency for another asset
-	* If you collect this item, two Sankara stones, and the Cross of Coronado, you can buy the ark of the covenant:
-		* Ark of the covenant is rare. It is 1 of 1.
-		* Ark of the covenant is legendary.
-		* Ark of the covenant gives you lifetime access to Indiana Jones World.
-		* Ark of the covenant has rules; 20% of the resale price goes to Indiana Jones World.
+  * https://indianajones.fandom.com/wiki/Crystal_Skull_of_Akator;
+  * press reports that this artefact could be worth $X.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 3]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
 Some examples of how different types of digital assets with different attributes, rules and interactions may be 
-manifested are shown below.
+manifested are provided here:
 
 #### Crystal Skull of Akador
 
-* Is rare
-	* Is 1 of 5
+* Is rare, it is 1 of 5.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md (hunk 2 above)
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 4]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
-	* If you collect this item, two Sankara stones, and the Cross of Coronado, you can buy the ark of the covenant:
-		* Ark of the covenant is rare. It is 1 of 1.
-		* Ark of the covenant is legendary.
-		* Ark of the covenant gives you lifetime access to Indiana Jones World.
-		* Ark of the covenant has rules; 20% of the resale price goes to Indiana Jones World.
+  * https://indianajones.fandom.com/wiki/Crystal_Skull_of_Akator;
+  * press reports that this artefact could be worth $X.
+
+* Drives collectability.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 5]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+* Drives advance purchase, e.g. if you are one of the first 100,000 people to buy tickets to Indiana Jones World, you 
+have a chance of winning this 
+  1 of 5 artefact.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 6]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+* Has superpowers and utility, e.g. if you have this item while visiting Indiana Jones World, you get to skip the line 
+three times.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 7]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+* Is a contingency for another asset, e.g. if you collect this item, two Sankara stones and the Cross of Coronado, you 
+  can buy the ark of the covenant:
+  - Ark of the covenant is rare. It is 1 of 1.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 8]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+  - Ark of the covenant is legendary.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 9]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+  - Ark of the covenant gives you lifetime access to Indiana Jones World.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 10]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
-* Has no utility
-* Has no rules
+- Is not rare, it is 1 of 100,000.
+- Is legendary - https://www.youtube.com/watch?v=HK6B2da3DPA.
+- Drives collectability, it is part of a series of bats from famous batsmen.
+- Can be combined with other assets, e.g. be one of the first 10 people to combine six bats to turn this asset into a 
+One Day International (ODI) century bat:
+  - ODI century bats are rare, they are 1 of 10;
+  - ODI century bats are legendary.
+
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 11]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+* Has no superpowers.
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
- Has no superpowers.
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 12]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+* Has no utility.
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
- Has no utility.
```

File: RFC/src/RFC-1000_TariUseCases.md [hunk 13]
```diff
@@ -102,96 +106,108 @@ Digital asset tokens may influence the following behavioral types:
...
+* Has no rules.
```
Comment (hansieodendaal) [inline RFC/src/RFC-1000_TariUseCases.md]:
```suggestion
- Has no rules.
```

**Comment by anselld** [comment]
Hi @hansieodendaal, thanks very much for fixing the spacing issues, which I committed. I checked after pulling and verifying on my local mdbook.

**Comment by hansieodendaal** [comment]
Thanks @anselld, please combine commits

**Comment by CjS77** [comment]
Merged.

**Pull request closed without merging**